src/pages/ModelizerPage.vue

```diff
@@ -2,13 +2,17 @@
   <q-page
     class="modelizer-page"
     data-cy="modelizer-page"
-    >
-    <modelizer-navigation-bar
-      :projectName="projectName"
-      :viewType="viewType"
-      @changeView="changeView"/>
-    <modelizer-model-view v-show="viewType === 'model'" />
-    <modelizer-text-view v-show="viewType === 'text'" />
+  >
+    <q-layout>
+      <modelizer-navigation-bar
+        :projectName="projectName"
+        :viewType="viewType"
+        @changeView="changeView"
+      />
+      <components-panel v-show="viewType === 'model'" />
```

Comment:
prefere to have: 
```
<template v-show="viewType === 'model'" >
(...)
</template>
<template v-show="viewType === 'text'" >
(...)
</template>
```

Reply:
Do you mean to not use the self closing tag syntax ?

Reply:
I want that each component of the ModelView are in a `template` tag, same for the TextView :)

Reply:
[`v-show` directives can't be put on `template`s](https://eslint.vuejs.org/rules/valid-v-show.html)